### ISSUE TITLE
New functionality was added Linked Process and Linked Parameters

### ISSUE THREAD

**Issue by e-Evolution** [adempiere/adempiere#808]
Steve can you explain how work this new functionality.

![captura de pantalla 2017-03-03 a la s 14 40 02](https://cloud.githubusercontent.com/assets/786968/23568255/5255e8da-001f-11e7-9d47-8379e8138e53.png)

![captura de pantalla 2017-03-03 a la s 14 40 25](https://cloud.githubusercontent.com/assets/786968/23568266/5f60f5a6-001f-11e7-8bb8-036db332bc19.png)

**Comment by JobrianTrinidad** [comment]
Hi Victor,

This functionality only on works on WebUI, on HTML report view.  This function allows you to execute another process within the report, using the field as a parameter  (e.g. in Inventory valuation report, you have Product and Qty, a Product in the report could have a hyperlink to execute a report for Inventory Transaction).

I'm not sure if you have access to our redmine, but this is how it should look.

![1443_linkedprocess](https://cloud.githubusercontent.com/assets/22760273/23591280/9ca38af2-0252-11e7-95f2-99480a7a7709.png)

![1443_linkedprocesspara1](https://cloud.githubusercontent.com/assets/22760273/23591283/a853120a-0252-11e7-8052-5989cd7a38ac.png)

Link_ID is a reserved word pertaining to the value of the link column (e.g. M_Production_ID = 1000001)

![1443_linkedprocesspara2](https://cloud.githubusercontent.com/assets/22760273/23591285/adb971e4-0252-11e7-909d-4cd8ce87569f.png)

I hope this helps.

Regards,

Jobrian

**Comment by marcalwestf** [comment]
@JobrianTrinidad thanks for the explanation. 
I believe, the process is displayed as a link, doesn't it?

Next step would be to test it.

**Comment by JobrianTrinidad** [comment]
@marcalwestf 
It will add a new item to the menu [Window, Report, Process*]
As shown below.

![menuitem](https://cloud.githubusercontent.com/assets/22760273/23770036/587c34c4-0576-11e7-9adc-a33c706c01d1.jpg)

**Comment by marcalwestf** [comment]
this hasn't been implemented.

**Comment by marcalwestf** [comment]
here also, these are the changes in Adaxa repo:
![20170707_dynamiczoom_02](https://user-images.githubusercontent.com/1789408/27973430-411a3d22-6317-11e7-9dd3-2826b3074a53.png)

whereas in Adempiere repo only xml file and model classes have been integrated.
The other classes are missing.

**Comment by marcalwestf** [comment]
Now, the _Table_ window is defined this way:
![20170707_ad_02](https://user-images.githubusercontent.com/1789408/27981280-4bce2c94-6347-11e7-9b6c-1e8e8194ce2b.png)

And the _Table_ window looks as follows:
![20170707_ad_03](https://user-images.githubusercontent.com/1789408/27981289-64fc9976-6347-11e7-8b22-a48fac6c5081.png)

**Comment by yamelsenih** [comment]
Hi, any update for it?

**Comment by e-Evolution** [comment]
this functionality was disable , because not work rightly, but I think that Steven @adaxa-steven can give more info , this was take from Adaxa features list ,but is pending your integration.

**Comment by yamelsenih** [comment]
@adaxa-steven have you a pull request for it?

**Comment by adaxa-steven** [comment]
@yamelsenih  .. too technical for me.. I think it is a lot more manual process to get from our Hg repository?.
can do a screen share and show how it works and you can decide if it improves Adempiere .. we find it very useful.  Let me know if you want to see how it works.
/steven

**Comment by yamelsenih** [comment]
Yes off course, please tell me about it and if you have a commit or change to be included please give me it
